apis/go.mod

```diff
@@ -8,34 +8,34 @@ require (
+	github.com/openstack-k8s-operators/horizon-operator/api v0.3.1-0.20240411120107-bc969613d25f
 	github.com/openstack-k8s-operators/infra-operator/apis v0.3.1-0.20240409161822-d0ab878db2d8
 	github.com/openstack-k8s-operators/ironic-operator/api v0.3.1-0.20240408054123-cb7b79a22b47
-	github.com/openstack-k8s-operators/keystone-operator/api v0.3.1-0.20240402154848-e5f862707f49
-	github.com/openstack-k8s-operators/lib-common/modules/common v0.3.1-0.20240409150851-9a336392d3b2
-	github.com/openstack-k8s-operators/lib-common/modules/storage v0.3.1-0.20240409150851-9a336392d3b2
+	github.com/openstack-k8s-operators/keystone-operator/api v0.3.1-0.20240412212552-850399d0605c
+	github.com/openstack-k8s-operators/lib-common/modules/common v0.3.1-0.20240412091425-bb628ded5eb8
+	github.com/openstack-k8s-operators/lib-common/modules/storage v0.3.1-0.20240412091425-bb628ded5eb8
 	github.com/openstack-k8s-operators/manila-operator/api v0.3.1-0.20240403143013-74e2f3cc4015
-	github.com/openstack-k8s-operators/mariadb-operator/api v0.3.1-0.20240409152037-91b5c56dfc4a
-	github.com/openstack-k8s-operators/neutron-operator/api v0.3.1-0.20240409132013-b62679976372
-	github.com/openstack-k8s-operators/nova-operator/api v0.3.1-0.20240404163736-a2d67e46f9f2
+	github.com/openstack-k8s-operators/mariadb-operator/api v0.3.1-0.20240411135034-a77c10351c47
+	github.com/openstack-k8s-operators/neutron-operator/api v0.3.1-0.20240411120933-2fa11969312b
+	github.com/openstack-k8s-operators/nova-operator/api v0.3.1-0.20240415072306-b848abde3433
 	github.com/openstack-k8s-operators/octavia-operator/api v0.3.1-0.20240408184306-f4d50944f99d
-	github.com/openstack-k8s-operators/ovn-operator/api v0.3.1-0.20240411042149-e75b363fc851
+	github.com/openstack-k8s-operators/ovn-operator/api v0.3.1-0.20240412212308-52c4fc7de5a4
 	github.com/openstack-k8s-operators/placement-operator/api v0.3.1-0.20240404140050-69252e99daaf
-	github.com/openstack-k8s-operators/swift-operator/api v0.3.1-0.20240410220810-833995fece81
-	github.com/openstack-k8s-operators/telemetry-operator/api v0.3.1-0.20240409214006-1fe1e9980488
+	github.com/openstack-k8s-operators/swift-operator/api v0.3.1-0.20240412224825-4de3d73ff582
+	github.com/openstack-k8s-operators/telemetry-operator/api v0.3.1-0.20240411074805-079ca1366960
 	github.com/rabbitmq/cluster-operator/v2 v2.6.0
 	github.com/rhobs/obo-prometheus-operator/pkg/apis/monitoring v0.69.0-rhobs1 // indirect
 	github.com/rhobs/observability-operator v0.0.28 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
 	go.uber.org/zap v1.27.0 // indirect
 	golang.org/x/exp v0.0.0-20240213143201-ec583247a57a // indirect
 	golang.org/x/tools v0.18.0 // indirect
-	k8s.io/api v0.28.8
-	k8s.io/apimachinery v0.28.8
-	k8s.io/client-go v0.28.8
-	sigs.k8s.io/controller-runtime v0.16.5
+	k8s.io/api v0.29.0
+	k8s.io/apimachinery v0.29.1
+	k8s.io/client-go v0.29.0
+	sigs.k8s.io/controller-runtime v0.17.0
```

Comment:
should not have bumped to this version

Reply:
Yeah, how did that happen with [1] and [2]?

[1] https://github.com/openstack-k8s-operators/renovate-config/blob/5417bc2dccb61e89a416ca64fe70b0a21776c925/default.json5#L29-L37
[2] https://github.com/openstack-k8s-operators/renovate-config/blob/5417bc2dccb61e89a416ca64fe70b0a21776c925/default.json5#L54-L60